apps/api/webpack.config.js

```diff
@@ -1,12 +1,5 @@
 const { composePlugins, withNx } = require('@nx/webpack');
-const CopyPlugin = require('copy-webpack-plugin');
```

Comment:
can you run npm remove on this package please

Reply:
Done. Thanks for the catch!